### PR TITLE
fix(DB/conditions): Fix Quest Kibler's Exotic Pets

### LINKED ISSUE #582: DB/Quest: Kibler's Exotic Pets
**Description**: This is the kind of bug that makes me think the Sunwell guys didn't do as much as they claimed, either that or they did not put a focus on vanilla quests. Either way, this quest can be completed simply by clicking on the worg pup cage quest item inside the instance. I happened to click it in the Skitterweb Tunnels where all of the spiders are and to my surprise I captured a worg pup. /facepalm

**Expected behavior:** 
You should only be able to use the carrier inside the worg den where the mother worg resides. There are many small pups running around here.

**Steps to reproduce the problem:**
Use the worg pup carrier in Blackrock Spire.

**Branch:**
Master

**Commit Hash:** 
c15f15ebb0a113a5cf20fc11afb89b2bcff9ec6d

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/48656423-db-quest-kibler-s-exotic-pets?utm_campaign=plugin&utm_content=tracker%2F40032087&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F40032087&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by KaizerAce**
You were able to complete the quest anywhere without the required creature being around you.

## CHANGES PROPOSED:
-  Add a condition to the spell

## ISSUES ADDRESSED:
- Closes #582

## TESTS PERFORMED:
- Executed SQL
-Tried to complete quest anywhere in the world
- Teleported near needed creature and watched it work

## HOW TO TEST THE CHANGES:

.quest add 4729
attempt to use added item "it should not work"

now do this:
.go c 40169
use item again

## KNOWN ISSUES AND TODO LIST:

- [ ]
- [ ] 

## Target branch(es):
- [x] Master

 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1605488437069394200.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1605488437069394200');
+
+DELETE FROM `conditions` WHERE `SourceTypeOrReferenceId` = 17 AND `SourceEntry` = 15998;
+INSERT INTO `conditions` (`SourceTypeOrReferenceId`, `SourceGroup`, `SourceEntry`, `SourceId`, `ElseGroup`, `ConditionTypeOrReference`, `ConditionTarget`, `ConditionValue1`, `ConditionValue2`, `ConditionValue3`, `NegativeCondition`, `ErrorType`, `ErrorTextId`, `ScriptName`, `Comment`) VALUES 
+(17, 0, 15998, 0, 0, 29, 0, 10221, 3, 0, 0, 0, 0, '', Add creature requirement to Empty Worg Pup Cage);
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1605488437069394200.sql:5]:
```suggestion
(17, 0, 15998, 0, 0, 29, 0, 10221, 3, 0, 0, 0, 0, '', 'Empty Worg Pup Cage requires Bloodaxe Worg Pup (3y)');
```

**Review by Kitzunu — CHANGES_REQUESTED** [review]
This will fix the sql error
And add a useful comment to it :)

**Comment by KaizerAce** [comment]
> This will fix the sql error
> And add a useful comment to it :)

Thank you, way better description lol

**Review by FrancescoBorzi — APPROVED** [review]

**Review by Si1ker — APPROVED** [review]
Tested ✔️ 

If no in range, cannot do that yet.
image

Now Im in range so I can
image

And... completed..
image

**Comment by FrancescoBorzi** [comment]
thank you @KaizerAce for this contribution and of course @Si1ker for the tests and screenshots!

**Pull request merged**